### ISSUE TITLE
Browser navigation in tutorials

### ISSUE THREAD

**Issue by rpruim** [rstudio/learnr#86]
It would be nice to be able to get completely out of a tutorial more easily.  As is, there are multiple ways to navigate within the pages of a tutorial, but no good way to get back to the page one was on before starting the tutorial.

Is there a way to do that given the current set-up?  I'm imagining that one might publish a page with links to several tutorials, and someone might get part way into one of the tutorials and like to get back to the referring page.

**Comment by jjallaire** [comment]
Yes, the back button works all the way through so just like navigating a
website you need to just keep hitting "back". I'd expect that tutorial
hosting service would implement some sort of meta-navigation along the
lines you envision.

**Comment by rpruim** [comment]
In the meantime, might it be possible to create a running footer somehow?  The tutorial author could put anything there, including a link to a web page with lists of tutorials, or the author's home page, or some other reference materials.

**Comment by jjallaire** [comment]
It's an R Markdown document so you can do whatever you want in the footer!

**Comment by rpruim** [comment]
I tried this, but the positioning is pretty bad with `after_body`.  Is there a different command to place this just below the previous/next page navigation?

**Comment by jjallaire** [comment]
I would just try to use CSS and JavaScript to get the content where you
need it (sorry I can't do more for you right now, in the middle of
completing some other work that is taking all of my time).
